css/main.css

```diff
@@ -39,13 +171,13 @@ body {
   position: relative;
   z-index: 1;
 }
-.login_button:hover {
+.login_button_sml:hover {
```

Comment:
Maybe login_button_sm?

Reply:
Hmm.. I changed it earlier, maybe then I had forgotten to push my version into Github.